tmt/steps/prepare/brand_new_allmighty_install/providers/__init__.py

```diff
@@ -0,0 +1,147 @@
+"""
```

Comment:
Regarding the name could we use something shorter than `brand_new_allmighty_install`? What about just `almighty`? I was also thinking whether the final name could possibly be `artifacts`? The main goal is to install various artifacts for testing. For now packages only, for the future container images were proposed... I'd say `artifacts` should be general enough to cover possible future extensions as well. Thoughts?

Reply:
:+1: for `artifact` name. Probably dropping the plural since we don't use it in other places. One maybe down-side to such a naming could be that it invites other people to hook into it and create their own artifacts, e.g. a rust crate artifact.

Reply:
Yeah, singular `artifact` would be better.